.travis.yml

```diff
@@ -30,11 +30,3 @@ install:
 script:
     - echo "Updating docs";
     - bash ./.ci_scripts/update_docs;
-
-# We store the files that are downloaded from continuum.io, but not the environments that are created.
-cache:
-    directories:
-      - $HOME/cache
-before_cache:
-  # Remove all untarred directories.
-  - find $CONDA_INSTALL_LOCN/pkgs/ -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
```

Comment:
Just curious why you removed caching? It was not working/worth it? Or it doesn't work with miniforge?

Reply:
I thought it wasn't working because travis-ci downloaded a bunch of files in https://travis-ci.com/github/conda-forge/blog/builds/196075727

Reply:
Possible, that is an old syntax and travis changed quite a bit since.